### ISSUE TITLE
Installation on a macOS Catalina 10.15.7

### ISSUE THREAD

**Issue by desandy** [containers/bubblewrap#396]
Does this package support the macOS operating system?

I have tried running configure and/or autogen, both result in the following:

checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports the include directive... yes (GNU style)
checking whether make supports nested variables... yes
checking whether UID '502' is supported by ustar format... yes
checking whether GID '20' is supported by ustar format... yes
checking how to create a ustar tar archive... gnutar
checking dependency style of gcc... gcc3
checking whether to enable maintainer-specific portions of Makefiles... yes
checking whether make supports nested variables... (cached) yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for gcc... (cached) gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to accept ISO C89... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking sys/capability.h usability... no
checking sys/capability.h presence... no
checking for sys/capability.h... no
configure: error: *** POSIX caps headers not found

I have not been able to find a work around for this issue yet.

Things I have tried:

I have tried both ./configure and ./autogen.sh

I have tired setting the CPATH variable.

I have install xcode-select --install

sudo ln -s /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/* /usr/local/include

Any help would be appreciated.

**Comment by smcv** [comment]
> Does this package support the macOS operating system?

No. It uses Linux namespaces, which are specific to the Linux kernel.

**Comment by akavel** [comment]
I don't know much about either of the tools, and arrived here somewhat accidentally, but apparently [there's something called `sandbox-exec` on OSX](https://discuss.ocaml.org/t/ann-first-release-of-bwrap/2835#post_4), which may have at least some features in common with bubblewrap, I presume.
